package.json

```diff
@@ -10,8 +10,16 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@ant-design/icons": "^5.2.5",
     "@grapesjs/react": "^1.0.0",
+    "antd": "^5.8.4",
+    "axios": "^1.5.0",
+    "body-parser": "^1.20.2",
+    "cors": "^2.8.5",
+    "express": "^4.18.2",
+    "express-ejs-layouts": "^2.5.1",
     "grapesjs": "^0.21.4",
+    "mongoose": "^7.4.5",
```

Comment:
Let’s please uninstall this package, since we’re not going to use on the client side🙈